source/core/src/main/com/csse3200/game/components/player/PlayerCoinDisplay.java

```diff
@@ -63,6 +69,31 @@ private void updateCoinLabel() {
         coinLabel.setText("Coins: x" + coinsComponent.getCoins());
     }
 
+    /**
+     * When a player successfully defeats an animal, the
+     * amount of coins earned is displayed on the screen for few seconds
+     *
+     * @param coins the amount of coins an animal gave from defeating
+     */
+    private void showCoins(int coins) {
+        ServiceLocator.getResourceService().playSound("sounds/collectCoin.mp3");
+        String text = String.format("+%d coins added", coins);
+        Label coinText = new Label(text, skin, "small");
+        Table table = new Table();
+        table.top().padTop(150f);
+        table.setFillParent(true);
+        table.row().padTop(5f);
+        table.add(coinText).padTop(5f);
```

Comment:
you can probably just sum these padding vals up and just add it in one place, this should work too